CONTRIBUTING.rst

```diff
@@ -0,0 +1,132 @@
+.. highlight:: shell
+
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+For a high-level overview of the philosophy of contributions to PlanetaryPy,
+which applies to this project, please see
+https://github.com/planetarypy/TC/blob/master/Contributing.md.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/somedev/package_template_build/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug" and "help
+wanted" is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "enhancement"
+and "help wanted" is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+package_template_build could always use more documentation, whether as part of the
+official package_template_build docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at https://github.com/somedev/package_template_build/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `package_template_build` for local development.
+
+1. Fork the `package_template_build` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/package_template_build.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+
+    $ mkvirtualenv package_template_build
+    $ cd package_template_build/
+    $ python setup.py develop
```

Comment:
This needs to change to `pip install -e .` actually, as using `pip` is the favored method over python setup.py due to its cleaner uninstall.

Reply:
Fair enough.